Makefile

```diff
@@ -51,7 +51,7 @@ cover:
 	go tool cover -html=coverage.out
 
 e2e: generate
-	./hack/e2e.sh
+	./hack/e2e.sh "${SUITE}"
```

Comment:
This script needs to run all e2e tests that we have so we should not make this change. Otherwise, we need to update the way we run e2e tests in openshift/release.

Reply:
@kargakis as you can see in `hack/e2e.sh` the current behavior is maintained: an empty $SUITE variable will run all tests, this is just to override that behavior.

Reply:
ack